include/irr/asset/IDescriptorSetLayout.h

```diff
@@ -0,0 +1,45 @@
+#ifndef __IRR_I_DESCRIPTOR_SET_LAYOUT_H_INCLUDED__
+#define __IRR_I_DESCRIPTOR_SET_LAYOUT_H_INCLUDED__
+
+#include "irr/asset/ShaderCommons.h"
+#include "irr/asset/SSamplerParams.h"
+
+namespace irr { namespace asset
+{
+
+enum E_DESCRIPTOR_TYPE
+{
+    EDT_SAMPLER = 0,
+    EDT_COMBINED_IMAGE_SAMPLER = 1,
+    EDT_SAMPLED_IMAGE = 2,
```

Comment:
cant support image/texture without smapler

Reply:
@Crisspl TODO

Reply:
you mean remove this enum val?

Reply:
remove `EDT_SAMPLER` and `EDT_SAMPLED_IMAGE`

Reply:
resolved